lua/neotest-rspec/init.lua

```diff
@@ -23,23 +24,7 @@ end
 ---@param position_id string
 ---@return string
 local function form_treesitter_id(position_id)
-  return position_id
-    -- :gsub('<NS>.-</NS> ', '', 1) -- Remove the filename from the id
-    :gsub('<NS>type:.-</NS> ', '') -- Remove any 'type: xx ' strings
-    :gsub(' <NS>"#', '#') -- Weird edge case
-    :gsub('<TS>should be_empty</TS>', 'is expected to be empty') -- RSpec's one-liner syntax
-    :gsub('<TS>is_expected.to be_empty</TS>', 'is expected to be empty') -- RSpec's one-liner syntax
-    :gsub("<NS>'", '')
-    :gsub("'</NS>", '')
-    :gsub('"</NS>', '')
-    :gsub('<NS>"', '')
-    :gsub("<TS>'", '')
-    :gsub("'</TS>", '')
-    :gsub('<TS>"', '')
-    :gsub('"</TS>', '')
-    :gsub('<NS>', '')
-    :gsub('<NS>', '')
-    :gsub('</NS>', '')
+  return utils.form_treesitter_id(position_id)
```

Comment:
I agree with abstracting to a seperate lua file. We could then just remove the local function alltogether and call `utils.form_treesitter_id` in the `NeotestAdapter.discover_positions` method.

Also think we should move `parse_json_output` to the utils file too.

Reply:
Good point that the local doesn't add much value anymore.

I was also thinking that extracting `parse_json_output` would be useful.